languages/csharp/exercises/concept/numbers/.docs/instructions.md

```diff
@@ -2,9 +2,11 @@ In this exercise you'll be writing code to analyze the production of an assembly
 
 At its lowest speed (`1`), `221` cars are produced each hour. The production increases linearly with the speed. So with the speed set to `4`, it should produce `4 * 221 = 884` cars per hour. However, higher speeds increase the likelihood that faulty cars are produced, which then have to be discarded. The following table shows how speed influences the success rate:
```

Comment:
Is 0 now the lowest speed, instead of 1? Or is this an intentional ask of the student to implicitly recognise that 0 is "no speed" ?

Reply:
I've removed it. Might indeed cause more confusion.